internal/local/environment_test.go

```diff
@@ -0,0 +1,29 @@
+package local
+
+import (
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestLoadProfiles(t *testing.T) {
+	t.Run("should return a list of profile names, file paths and api keys", func(t *testing.T) {
+		profile, _ := mock.NewProfileFromTmpDir(t, "temp")
+
+		profile.SetCredentials(user.Credentials{PublicAPIKey: "abcde", PrivateAPIKey: "*****"})
```

Comment:
less meaningful here but an equal sentiment about using `some-super-secret-key` value instead of simply asterisks for the value here